src/Model/Export/Catalog/ExportPreviewProducts.php

```diff
@@ -10,6 +10,7 @@
 use Magento\Framework\App\RequestInterface;
 use Traversable;
 
+#[\AllowDynamicProperties]
```

Comment:
Rather than allowing dynamic properties I would advice to specifically define the `$entityId` member variable instead.

Reply:
Your are right. Thanks 👍